tools/cldr-apps/js/src/esm/cldrAjax.js

```diff
@@ -42,30 +62,36 @@ function doFetch(resource, init) {
  * }
  */
 function sendXhr(xhrArgs) {
+  const options = newOptionsFromXhrArgs(xhrArgs);
+  const request = new XMLHttpRequest();
+  request.open(options.method, xhrArgs.url);
+  request.responseType = options.handleAs ? options.handleAs : "text";
+  request.timeout = xhrArgs.timeout ? xhrArgs.timeout : 0;
+  addSessionHeaderIfApi(request, xhrArgs.url);
```

Comment:
not all APIs require a session header.

Reply:
is there any harm in setting it anyway?

Reply:
No, probably not unless it was an explicit 'log out' endpoint, which is already handled separately.